docs/_docs/analysis.md

```diff
@@ -228,14 +228,17 @@ The input keywords are:
 
 ## Charge Fluctuations
 
-Calculates the average charge, the standard deviation of the charge, and the most 
-probable species (atom name) of atomic indexes undergoing charge fluctuations.
+For a given molecule, this calculates the average charge, the standard deviation of the charge, 
+and the most probable species (atom name) averaging over all molecular groups.
+The structure of one of the molecular groups with average charges and most probable 
+atomic species is saved in PQR format.
```

Comment:
Use instead a `pqrfile` keyword to let the user decide the name and if a pqr file should be saved?